### PR TITLE
API-32311 establish poa oas updates

### PR DESCRIPTION
**Pull request by rockwellwindsor-va**
## Summary
* Updates OAS text
* Updates Swagger Docs
* Adds 2122, 2122a, 2122/validate, 2122a/validate and /power-of_attorney/{id} endpoint to docs

## Related issue(s)
[API-32311](https://jira.devops.va.gov/browse/API-32311)

## Testing done
Documentation updates , so no testing

## Screenshots
Screenshot 2024-03-19 at 8 34 16 AM

POA V2 docs:

Screenshot 2024-03-25 at 3 05 54 PM

Endpoints added in Swagger (new routes at bottom)

Screenshot 2024-03-26 at 10 53 25 AM

## What areas of the site does it impact?
	modified:   modules/claims_api/app/swagger/claims_api/description/v2.md
	modified:   modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json
	modified:   modules/claims_api/app/swagger/claims_api/v2/production/swagger.json

## Acceptance criteria

- [ ]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [x]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: modules/claims_api/app/swagger/claims_api/description/v2.md [hunk 1]
```diff
@@ -6,14 +6,13 @@ The Benefits Claims API Version 2 lets internal consumers:
 -   Automatically establish an Intent To File (21-0966) in VBMS
 -   Automatically establish a disability compensation claim (21-526EZ) in VBMS
 -   Digitally submit supporting documentation for disability compensation claims
--   Retrieve the active Power of Attorney for a Veteran
+-   Retrieve the active Power of Attorney organization of individual with power of attorney for a Veteran
+-   Automatically establish a power of attorney appointment in VBMS for an accredited organization (VA Form 21-22).
+-   Automatically establish a power of attorney appointment in VBMS for an accredited individual (VA Form 21-22a).
 
 Additional functionality will be added over time.
```
Comment (mchristiansonVA) [inline modules/claims_api/app/swagger/claims_api/description/v2.md:13]: This line is crossed-out in the ticket...looks like it should be removed?

Comment (rockwellwindsor-va) [reply modules/claims_api/app/swagger/claims_api/description/v2.md:13]: Darn, yes that is correct, I re compiled the docs and did not push that up yet

Comment (rockwellwindsor-va) [reply modules/claims_api/app/swagger/claims_api/description/v2.md:13]: Ya good catch missed that line, fixing now

**Review by mchristiansonVA — APPROVED** [review]
LGTM!

**Review by stiehlrod — APPROVED** [review]

**Review by FonzMP — APPROVED** [review]
:shipit:

**Pull request merged**